test/Sandbox.jl

```diff
@@ -307,15 +307,20 @@ for executor in all_executors
                     ro_mappings["/etc/resolv.conf"] = resolv_conf
                 end
 
+                # See https://github.com/staticfloat/Sandbox.jl/issues/77
+                persist = executor <: UnprivilegedUserNamespacesExecutor && !Sandbox.probe_xattr("user")
+                @show persist
                 config = SandboxConfig(
                     ro_mappings,
-                    Dict("/tmp/rw_dir" => rw_dir),
+                    Dict("/tmp/rw_dir" => rw_dir);
+                    persist,
                 )
 
                 # Test that we can use `wget` (née busybox) within the alpine rootfs to download something
                 socrates_url = "https://github.com/staticfloat/small_bin/raw/master/socrates.tar.xz"
                 socrates_hash = "61bcf109fcb749ee7b6a570a6057602c08c836b6f81091eab7aa5f5870ec6475"
                 with_executor(executor) do exe
+                    persist && (exe.persistence_dir = @get_scratch!("tmpfs_xattr_workaround"))
```

Comment:
I've put this year, but probably should go to https://github.com/staticfloat/Sandbox.jl/blob/f484672e5b5f3aff2e30b56821c18fd3d3e5d581/src/UserNamespaces.jl#L169 instead?